### DISCUSSION TITLE
Errors in practical 0301

### DISCUSSION THREAD

**Discussion by Nambunga** [IBAHCM/RPiR#226] [Archive] [answered]
Hi! @soniamitchell @richardreeve I tried add up new demo in package as per instruction provided, however, once tried checking if this works, I got the following message:
"Error in demo("d0302_stochastic_birth_death.R", package = "NambungaSeries03") : 
  No demo found for topic ‘d0302_stochastic_birth_death.R’".

Could you please help me the way to parse this kind of error?
Thanks

**Comment by soniamitchell** [comment, marked as answer]
Hey @Nambunga, turns out there was a typo in the Practical, it should be 
```
demo("d0302_stochastic_birth_death", package = "NambungaSeries03")
```
Should be updated now 👍

**Comment by Nambunga** [comment]
Thank you, it’s working

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
